getting_started/first_2d_game/03.coding_the_player.rst

```diff
@@ -476,14 +476,28 @@ Inspector tab to see the list of signals the player can emit:
 Notice our custom "hit" signal is there as well! Since our enemies are going to
 be ``RigidBody2D`` nodes, we want the ``body_entered(body: Node2D)`` signal. This
 signal will be emitted when a body contacts the player. Click "Connect.." and
-the "Connect a Signal" window appears. We don't need to change any of these
-settings so click "Connect" again. Godot will automatically create a function in
-your player's script.
+the "Connect a Signal" window appears.
+
+Godot will attempt to create a function with that exact name directly in script
```

Comment:
```suggestion
Godot will create a function with that exact name directly in script
```
I think leaving out the attempt to is best. Yes there is currently the bug but you're not exactly doing a coin toss when using the editor, it just works. And the warning directly below it will be clearly visible to people